frontend/src/app/modules/group/group-list/group-list.component.html

```diff
@@ -7,14 +7,66 @@
   <app-loading-spinner size="20px"></app-loading-spinner>
 </ng-template>
 <ng-template #loaded>
+  <div *ngIf="
+          groupInvites?.length;
+          then existInvites;
+        ">
+
+  </div>
   <div *ngIf="
       groups?.length;
       then existGroups;
       else emptyGroups
     ">
   </div>
+
+  <ng-template #existInvites>
+    <div class="container for-span d-flex flex-row my-3">
+      <span class="my-auto title">Group invitations</span>
+    </div>
+    <div class="container" *ngIf="groupInvites">
+      <div class="row my-3 border rounded" *ngFor="let group of groupInvites">
+        <div class="col-md-3 mx-0 px-10 col-sm-3 col-xs-12 my-auto row">
+          <div class="col-md-12 col-sm-12 text-dark">
+            <i *ngIf="!group.isPublic" class="fas fa-lock" [routerLink]="['../../groups', group?.id, 'projects']"></i>
```

Comment:
It's better to use full route instead of relative path when working with different modules